rest/utilities_testing.go

```diff
@@ -736,9 +742,21 @@ func NewBlipTester(tb testing.TB) (*BlipTester, error) {
 	return NewBlipTesterFromSpec(tb, defaultSpec)
 }
 
+func NewBlipTesterAvoidRTClose(tb testing.TB, spec *BlipTesterSpec) (blipTester *BlipTester, err error) {
```

Comment:
As discussed, it would be nice to avoid future test bugs by removing the ability to specify a restTester in BlipTesterSpec.  This would force all callers who want to bring their own restTester to call this API (which you could probably rename to '..WithRT').